skl2onnx/operator_converters/ada_boost.py

```diff
@@ -356,13 +356,13 @@ def convert_sklearn_ada_boost_regressor(scope, operator, container):
                        median_idx_name,
                        name=scope.get_unique_operator_name('ArgMin'), axis=1)
     container.add_node(
-        'ArrayFeatureExtractor', [sorted_indices_name, median_idx_name],
-        median_estimators_name, op_domain='ai.onnx.ml',
-        name=scope.get_unique_operator_name('ArrayFeatureExtractor'))
+        'GatherElements', [sorted_indices_name, median_idx_name],
```

Comment:
What does GatherElements do? Is this a newly added op? I don't see it here: https://github.com/onnx/onnx/blob/master/docs/Operators-ml.md

Reply:
Yes it is here: https://github.com/onnx/onnx/blob/master/docs/Operators.md#GatherElements

cumsum was added too so I'll use it too.

Reply:
That means it's available in version 11 of the default onnx operator set. You should add version=11 and remove op_domain(which is wrongly set to ai.onnx.ml). Were you able to test this?

Reply:
Fixed. I implemented a version of gatherelements with other ONNX operators.